apps/api/pages/api/bookings/_post.ts

```diff
@@ -45,29 +41,34 @@ import { defaultResponder } from "@calcom/lib/server";
  *                 type: string
  *                 format: date-time
  *                 description: 'End time of the Event'
- *               name:
- *                 type: string
- *                 description: 'Name of the Attendee'
- *               email:
- *                 type: string
- *                 format: email
- *                 description: 'Email ID of the Attendee'
+ *               responses:
+ *                 type: object
+ *                 required:
+ *                    - name
+ *                    - email
+ *                    - metadata
+ *                    - location
+ *                 properties:
+ *                    name:
+ *                      type: string
+ *                      description: 'Name of the Attendee'
+ *                    email:
+ *                      type: string
+ *                      format: email
+ *                      description: 'Email ID of the Attendee'
+ *                    metadata:
```

Comment:
Is this field required? Maybe it shouldn't be?

Reply:
This was actually a good catch. I think on my call with @hariombalhara, I must have misinterpreted one of the changes. 🙌 
Resolved now